web-compat/web-compat-impl/build.gradle

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+android {
+    anvil {
+        generateDaggerFactories = true // default is false
+    }
+    namespace 'com.duckduckgo.webcompat.impl'
+}
+
+dependencies {
+    anvil project(path: ':anvil-compiler')
+    implementation project(path: ':anvil-annotations')
+    implementation project(path: ':di')
+    implementation project(path: ':common-utils')
+    implementation project(path: ':privacy-config-api')
+    implementation project(path: ':feature-toggles-api')
+    implementation project(path: ':app-build-config-api')
+    implementation project(path: ':web-compat-store')
+    implementation project(path: ':content-scope-scripts-api')
+
+    implementation Kotlin.stdlib.jdk7
+
+    implementation AndroidX.core.ktx
+
+    implementation AndroidX.appCompat
```

Comment:
Do you need all these dependencies? There's no activities or view models so I take we don't need them.